backend/types.bal

```diff
@@ -14,6 +14,7 @@
 // specific language governing permissions and limitations
 // under the License. 
 import web_app_marketplace.people;
+import ballerina/constraint;
```

Comment:
```suggestion
import web_app_marketplace.people;

import ballerina/constraint;
```

[1] https://learn-ballerina.github.io/best_practices/format_the_code.html#order-of-the-imports-in-a-bal-file